### PR TITLE
3 3 parameters selection and inverse model computing

### LINKED ISSUE #3: 3. Parameters Selection and Inverse Model Computing

### PR DESCRIPTION
**Pull request by JinsenLou**
Finish the inverse model computing module and the creation of GUI/ connection of model and UI.

### CODE REVIEW

File: Inverse Model Computing/soil_gui.py
Comment (wintelestr) [inline Inverse Model Computing/soil_gui.py:1]: It looks great, good logic and structure of codes.

**Review by wintelestr — COMMENTED** [review]
Great Job!

**Comment by dtdoris** [comment]
>  Code section for inversion is well commented and explained.

File: Inverse Model Computing/soil_gui.py
Comment (zhangqianalone) [reply Inverse Model Computing/soil_gui.py:1]: Good structure code with clear comments and the function is well built.

**Review by zhangqianalone — APPROVED** [review]
good structure codes with clear comments

**Comment by 9264Y** [comment]
The code has a very clean and organised structure.

**Pull request merged**